tests/libcxx/expected_results.txt

```diff
@@ -529,11 +529,6 @@ std/localization/locale.categories/category.monetary/locale.moneypunct/money_bas
 std/input.output/file.streams/fstreams/filebuf.virtuals/overflow.pass.cpp FAIL
 std/input.output/file.streams/fstreams/filebuf.virtuals/underflow.pass.cpp FAIL
 
-# STL bug: Our inheritance implementation is allowing this to compile when it shouldn't.

```

Comment:
We need to update `tests/libcxx/skipped_tests.txt` simultaneously. (The difference is that `expected_results.txt` is used by the GitHub test harness, while `skipped_tests.txt` is used by the MS-internal test harness. This is a known headache.)